db/Controllers/controller.js

```diff
@@ -57,6 +59,9 @@ exports.postCommentForArticleId = (req, res, next) => {
     res.status(201).send({ comment })
   })
   .catch((err) => {
+    if (err.code === '23503') {
+      res.status(404).send({status: 404, msg: 'ID not found'})
```

Comment:
We shouldn't be sending the response here, rather lets invoke our error handling middleware functions and allow them to decide on the response.